bentoctl_aws_ec2/utils/__init__.py

```diff
@@ -91,3 +96,14 @@ def push_docker_image_to_repository(
         docker_client.images.push(**docker_push_kwags)
     except docker.errors.APIError as error:
         raise Exception(f"Failed to push docker image {image_tag}: {error}")
+
+
+def get_tag_from_path(path):
```

Comment:
Use 
```
bento = Bento.from_fs( fs.open_fs(path) )
return bento.tag
```

Reply:
we have to load `Bento` from bentoml._internal.bento right, or will it be exposed somewhere else?

Reply:
from bentoml.bentos import Bento